ceph/ceph.py

```diff
@@ -544,10 +544,7 @@ def check_health(self, rhbuild, client=None, timeout=300):
         if not all(state in lines for state in valid_states):
             logger.error("Valid States are not found in the health check")
             return 1
-        if rhbuild.startswith('4'):
```

Comment:
ceph -s , output is different for rhceph 3.x and 4.x , those runs will fail if you edit this health function in framwork.Try to parse json output here and make it compatible with all suites or write a separate health check function for upgrade scenarios